cfn-resources/serverless-instance/mongodb-atlas-serverlessinstance.json

```diff
@@ -1,18 +1,9 @@
 {
+  "typeName": "MongoDB::Atlas::ServerlessInstance",
```

Comment:
[q] Just for my own knowledge, any reason the json got reformatted so much in these changes? MAkes it harder to review & wondering how we can avoid in the future.